modules/osdk-java-controller-reconcile-loop.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/java/osdk-java-tutorial.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-java-controller-reconcile-loop_{context}"]
+= Reconcile loop
```

Comment:
[Procedures](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) need to start with gerunds, have an intro statement, and contain the `.Procedure` section.

Reply:
Thanks - this is definitely a concept.